src/Acuminator/Acuminator.Tests/Helpers/EmbeddedFileDataAttribute.cs

```diff
@@ -5,31 +5,82 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
 using Xunit.Sdk;
+using Acuminator.Utilities;
 
 namespace Acuminator.Tests.Helpers
 {
 	public class EmbeddedFileDataAttribute : DataAttribute
 	{
-	    private readonly string[] _fileNames;
+		private const string TestsRoot = "Acuminator.Tests";
+		private const string SourcesPrefix = "Sources";
+		private const string DefaultPrefix = TestsRoot + "." + SourcesPrefix;
 
-	    public EmbeddedFileDataAttribute(params string[] fileNames)
-	    {
-	        _fileNames = fileNames;
-	    }
+		private readonly string _prefix;
+		private readonly string[] _fileNames;
+
+		public EmbeddedFileDataAttribute(string fileName, bool overloadResolution, [CallerFilePath]string testFilePath = null)
```

Comment:
@dropsonic we can omit it in the future when we will change folder structure. Otherwise the compiler can't decide between `EmbeddedFileDataAttribute(params string[] fileNames)` and `EmbeddedFileDataAttribute(string fileName, bool overloadFlag = true, [CallerMemberName]testFilePath = null )` in a situation like this:
`[EmbeddedFileData(@"MySourceFile.cs")]` 
If I remember correctly, the compiler will decide that this is call to the constructor with params

Reply:
@SENya1990 ok, we'll remove it since we don't need `params` overload at all